src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -618,7 +486,7 @@ private string PromptForResolvedPath(string pathFromLogFile)
             var openFileDialog = new OpenFileDialog
             {
                 Title = $"Locate missing file: {pathFromLogFile}",
-                InitialDirectory = Path.GetDirectoryName(CurrentSarifResult.LogFilePath),
+                InitialDirectory = this.sarifLocationTaggerService?.SelectedItem != null ? Path.GetDirectoryName(this.sarifLocationTaggerService.SelectedItem.LogFilePath) : null,
```

Comment:
>this.sarifLocationTaggerService?.SelectedItem [](start = 35, length = 45)

Pass in the SARIF error list item being used to this function. Don't use the selected item.